server/kit/kitserver.go

```diff
@@ -91,34 +91,38 @@ func NewServer(svc Service) *Server {
 
 	projectID := googleProjectID()
 	if projectID != "" {
-		svcName, svcVersion := "default", "v1.0"
+		svcName, svcVersion := "", ""
```

Comment:
Sorry, lil style nitpick...let's use default values and just do `var svcName, svcVersion string`